mreg_cli/utilities/api.py

```diff
@@ -591,20 +594,41 @@ def get_typed(
         return adapter.validate_json(resp.text)
 
 
+P = ParamSpec("P")
+
+
+def clear_cache(f: Callable[P, T]) -> Callable[P, T]:
+    """Clear the API cache after running the function."""
+
+    @wraps(f)
+    def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
+        result = f(*args, **kwargs)
+        try:
+            cache.evict("api")
+        except Exception as e:
+            logger.error(f"Error clearing cache: {e}")
+        return result
+
+    return wrapper
+
+
+@clear_cache
```

Comment:
Will we ever need multiple tagged caches? I can't see a reason for it, but maybe passing the tag is cleaner? Or name it `@clear_entire_cache`? Dunno.